src/main/java/loanbook/logic/commands/RemindCommand.java

```diff
@@ -31,49 +29,45 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Send a reminder email to the customer.\n"
             + "Parameter: "
-            + PREFIX_PASSWORD + "EMAILPASSWORD "
-            + PREFIX_NAME + "NAME "
-            + PREFIX_BIKE + "BIKE\n"
+            + PREFIX_EMAILPW + "EMAILPASSWORD "
+            + PREFIX_ID + "ID "
             + "Example: " + COMMAND_WORD + " "
-            + PREFIX_PASSWORD + "samplepassword "
-            + PREFIX_NAME + "John Doe "
-            + PREFIX_BIKE + "BIKE001";
+            + PREFIX_EMAILPW + "samplepassword "
+            + PREFIX_ID + "0 ";
 
     public static final String MESSAGE_SUCCESS = "Email sent!";
 
-    private final Name name;
-    private final Bike bike;
+    private final LoanId id;
     private final String emailPassword;
 
     /**
      * Creates an RemindCommand to send a reminder email to customer's {@code Email}
      * according to the {@code Name} and {@code BikeId} provided.
      */
-    public RemindCommand(String emailPassword, Name name, Bike bike) {
-        requireAllNonNull(emailPassword, name, bike);
-        this.name = name;
-        this.bike = bike;
+    public RemindCommand(String emailPassword, LoanId id) {
+        requireAllNonNull(emailPassword, id);
+        this.id = id;
         this.emailPassword = emailPassword;
     }
 
     @Override
     public CommandResult execute(Model model, CommandHistory history) throws CommandException {
         requireNonNull(model);
-        List<Loan> lastShownList = model.getLoanBook().getLoanList();
+        int maxId = model.getNextAvailableId().value - 1;
```

Comment:
Whoa, watch out here. The `getNextAvailableId()` method will increment the Loan ID counter.

I will admit that
- The method name might have been slightly misleading.
- I should have added a `peekNextAvailableId()`, or something like that.

What I would recommend is the following:

```java
public CommandResult execute(Model model, CommandHistory history) throws CommandException {
    Optional<Loan> optionalLoan = model.getLoanById(id);

    // Check if such a loan exists
    if (!optionalLoan.isPresent()) {
        throw new CommandException(/* A message like "A Loan with ID <insert ID here> does not exist in the Loan book." */);
    }

    Loan targetLoan = optionalLoan.get();

    // Check if the Loan has already been returned.
    if (targetLoan.getLoanStatus() == LoanStatus.RETURNED) {
        // Throw CommandException: Loan is done
    }

    // Then the rest of your method
}
```

Reply:
Fixed. Thank you so much :)